src/labels/CustomLabels.labels

```diff
@@ -2918,9 +2918,9 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Object groups description part 3</shortDescription>
-        <value>Let’s look at the two GAU Allocation 1 and GAU Allocation 2 Object Groups as an example. These object groups link directly to the Opportunity object. At times, 
-            you may want to assign two GAU Allocations to a single Opportunity. To achieve this, in your import file enter the Opportunity information in the Donations columns, 
-            then include the first GAU in the GAU Allocations 1 columns and the second GAU Allocation in the GAU Allocations 2 columns. Because of the GAU Allocation 1 and 
+        <value>Let’s look at the two GAU Allocation 1 and GAU Allocation 2 Object Groups as an example. These object groups link directly to the Opportunity object. At times,
```

Comment:
@ptewson-sfdo Just wanted to point these two labels out to you. We generally don't have trailing or leading spaces in custom labels. Meena's additional of two labels caused the trailing spaces on these two labels (bdiOMUIPageDescriptionPt3 and bdiOMUIThroughFieldHelp) to be removed. I'm not sure it will change anything in the UI, but it might be worth looking at where these are displayed either post merge (of this PR) or earlier to make sure that it doesn't change how the label text renders in the UI.

Reply:
@melissabarber